tasks/libs/ciproviders/github_api.py

```diff
@@ -89,6 +89,39 @@ def get_milestone_by_name(self, milestone_name):
                 return milestone
         return None
 
+    def get_branch_protection_checks(self, branch_name: str) -> list[str]:
+        """
+        Get the required checks for a given branch
+        """
+        branch = self.get_branch(branch_name)
+        if not branch:
+            raise Exit(color_message(f"Branch {branch_name} not found", Color.RED), code=1)
+        elif not branch.protected:
+            raise Exit(color_message(f"Branch {branch_name} doesn't have required checks", Color.RED), code=1)
+        protection = branch.get_protection()
+        return protection.raw_data['required_status_checks']['contexts']
```

Comment:
🥜 You have a data model you can use (rather than `getitem` from `raw_data`)
```suggestion
        return protection.required_status_checks.contexts
```

Reply:
Thanks, fixed with [35be02e](https://github.com/DataDog/datadog-agent/pull/31243/commits/35be02e3bfba8e775ed0836d67cb12ffd774f84d)!